packages/resource-deployment/scripts/function-app-create.sh

```diff
@@ -49,7 +51,7 @@ createAppRegistration() {
 
     appRegistrationName="allyappregistration-$resourceGroupName-$environment"
     echo "Creating a new AppRegistration with display name $appRegistrationName..."
-    clientId=$(az ad app create --display-name "$appRegistrationName" --query "appId" -o tsv)
+    clientId=$(az ad app create --display-name "$appRegistrationName" --query "appId" -o tsv) || true
```

Comment:
Should script exit if this line fails (which means ad app creation failed)?

Reply:
No, I don't think, we should be able to handle that manually since it's a small change, we already do that in our release pipelines, we all can talk also if you think we should